chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -53,24 +53,24 @@
 default[:openstack][:location] = 'C:\OpenStack'
 
 default[:openstack][:nova][:name] = "nova"
-default[:openstack][:nova][:version] = "2014.2.1"
-default[:openstack][:nova][:file] = "nova-2014.2.1.tar.gz"
+default[:openstack][:nova][:version] = "12.0.0.0rc2"
+default[:openstack][:nova][:file] = "nova-stable-liberty.tar.gz"
```

Comment:
hmm, there is no 12.0.0.0 in http://tarballs.openstack.org/neutron/ maybe neutron-2015.1.2.tar.gz ?